### PR TITLE
chore: add contributor spotlight markdown file

### LINKED ISSUE #263: chore: add spotlight contributors doc
As per our discussion we will have a folder called `spotlight-contributors` in this project as well as a Markdown file with information on our spotlight contributors. When nominating a new spotlight contributor, part of the process will be adding the relevant details to the Markdown document.

### PR DESCRIPTION
**Pull request by schalkneethling**
Adds spotlight contributor markdown file and a pull request template to use when nominating someone for a particular role.

fix #263

### CODE REVIEW

**Comment by bsmth** [comment]
Taking a look 👀

File: spotlight-contributors/index.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+
+### Carolyn Wu
+
+- [On GitHub](https://github.com/cw118)
+- [On MDN Web Docs](https://developer.mozilla.org/en-US/community/spotlight/cw118)
+
+### Michael Koch
+
+- [On GitHub](https://github.com/mikoMK)
+- [On MDN Web Docs](https://developer.mozilla.org/en-US/community/spotlight/mikoMK)
```
Comment (bsmth) [inline spotlight-contributors/index.md:15]:
```suggestion
- [On MDN Web Docs](https://developer.mozilla.org/en-US/community/spotlight/michael-koch)
```

**Review by bsmth — APPROVED** [review]
Looks good to me 👍🏻

File: .github/PULL_REQUEST_TEMPLATE/pull-request-template.md [hunk 2]
```diff
@@ -0,0 +1,18 @@
+# Nominating [GitHub username] as a [role]
```
Comment (dipikabh) [inline .github/PULL_REQUEST_TEMPLATE/pull-request-template.md:1]: Should we name this file `index.md`? Also we can follow lower case for the folder name. How about naming the folder something along these lines:
`.github/nomination_pull_request_template/index.md`

File: .github/PULL_REQUEST_TEMPLATE/pull-request-template.md [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+
+Thank you for taking the time to nominate a fellow contributor. Make sure you read our [list of prerequisites](https://developer.mozilla.org/en-US/docs/MDN/Community/Users_teams) for the relevant role before submitting a nomination.
```
Comment (dipikabh) [inline .github/PULL_REQUEST_TEMPLATE/pull-request-template.md:3]:
```suggestion
Thank you for taking the time to nominate a fellow contributor. Make sure you read our [list of prerequisites](https://developer.mozilla.org/en-US/docs/MDN/Community/Users_teams) for the relevant role before submitting a nomination.
Please submit information for the person you are nominating in the sections listed below.
```

File: .github/PULL_REQUEST_TEMPLATE/pull-request-template.md [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+
+## GitHub username (required)
+
+## GitHub profile link (required)
+
+## Why are you nominating this person? (required)
```
Comment (dipikabh) [inline .github/PULL_REQUEST_TEMPLATE/pull-request-template.md:9]:
```suggestion
## Reason for nominating this person (required)
```

File: .github/PULL_REQUEST_TEMPLATE/pull-request-template.md [hunk 5]
```diff
@@ -0,0 +1,18 @@
+# Nominating [GitHub username] as a [role]
+
+Thank you for taking the time to nominate a fellow contributor. Make sure you read our [list of prerequisites](https://developer.mozilla.org/en-US/docs/MDN/Community/Users_teams) for the relevant role before submitting a nomination.
+
+## GitHub username (required)
```
Comment (dipikabh) [inline .github/PULL_REQUEST_TEMPLATE/pull-request-template.md:5]:
```suggestion
## GitHub username of the nominee (required)
```

File: .github/PULL_REQUEST_TEMPLATE/pull-request-template.md [hunk 6]
```diff
@@ -0,0 +1,18 @@
...
+
+## GitHub profile link (required)
```
Comment (dipikabh) [inline .github/PULL_REQUEST_TEMPLATE/pull-request-template.md:7]:
```suggestion
## GitHub profile link of the nominee (required)
```

File: .github/PULL_REQUEST_TEMPLATE/pull-request-template.md [hunk 7]
```diff
@@ -0,0 +1,18 @@
...
+
+## Why are you nominating this person? (required)
+
+Tell us why you are nominating this person to become a spotlight contributor.
+
+## Supporting evidence (required)
```
Comment (dipikabh) [inline .github/PULL_REQUEST_TEMPLATE/pull-request-template.md:13]:
```suggestion
## Supporting evidence for nominating this person (required)
```

File: .github/PULL_REQUEST_TEMPLATE/pull-request-template.md [hunk 8]
```diff
@@ -0,0 +1,18 @@
...
+
+Provide evidence to support your nomination such as contributions and interactions with the community.
+Links to [commit logs](https://github.com/mdn/yari/commits?author=schalkneethling) in a repository are also appropriate.
```
Comment (dipikabh) [inline .github/PULL_REQUEST_TEMPLATE/pull-request-template.md:16]:
```suggestion
Provide evidence to support your nomination. The supporting information can include a list of their contributions and interactions with the community.
You can also provide links to [commit logs](https://github.com/mdn/yari/commits?author=schalkneethling).
```

File: spotlight-contributors/index.md [hunk 9]
```diff
@@ -0,0 +1,50 @@
+# MDN Web Docs Spotlight Contributors
```
Comment (dipikabh) [inline spotlight-contributors/index.md:1]:
```suggestion
# MDN Web Docs spotlight contributors
```

Also, how about the folder name `spolight_contributors`?

File: spotlight-contributors/index.md [hunk 10]
```diff
@@ -0,0 +1,50 @@
...
+
+For more details on our spotlight contributor program, see our [community documentation](https://developer.mozilla.org/en-US/docs/MDN/Community/Users_teams#spotlight_contributor).
+
+## Spotlight Contributors
```
Comment (dipikabh) [inline spotlight-contributors/index.md:5]:
```suggestion
## Archive of our spotlight contributors
```

File: spotlight-contributors/index.md [hunk 11]
```diff
@@ -0,0 +1,50 @@
...
+
+### Carolyn Wu
```
Comment (dipikabh) [inline spotlight-contributors/index.md:7]: Would it make sense to capture the month and year when the spotlight was on them?

```suggestion
### Carolyn Wu
August, 2022
```

**Review by dipikabh — CHANGES_REQUESTED** [review]
This looks great, @schalkneethling! Thank you. I have added a few suggestions and comments.

File: .github/PULL_REQUEST_TEMPLATE/pull-request-template.md (hunk 2 above)
Comment (schalkneethling) [reply .github/PULL_REQUEST_TEMPLATE/pull-request-template.md:1]: From what I can tell from the GitHub docs, pull request templates are still very primitive :) and so, you can only have one, really. They have some guidance as to naming, and this is one of the ways. It is also how we do it on other repos. On Yari with have it at the root: https://github.com/mdn/yari/tree/main/.github

File: spotlight-contributors/index.md (hunk 11 above)
Comment (schalkneethling) [reply spotlight-contributors/index.md:7]: Yeah, good idea. I was. thinking of doing that. I hope I can get the appropriate month from the GitHub repo :)

**Review by dipikabh — APPROVED** [review]
Thanks for the fixes, @schalkneethling!

**Comment by dipikabh** [comment]
For the two folder names, I guess I was following the conventions we've described in [Documentation conventions](https://github.com/mdn/content/blob/main/CONTRIBUTING.md#documentation-conventions). So we name the markdown file inside a folder as `index.md`, and we use lower case and underscores in the folder names.

**Comment by schalkneethling** [comment]
> For the two folder names, I guess I was following the conventions we've described in [Documentation conventions](https://github.com/mdn/content/blob/main/CONTRIBUTING.md?rgh-link-date=2022-12-05T19%3A35%3A31Z#documentation-conventions). So we name the markdown file inside a folder as `index.md`, and we use lower case and underscores in the folder names.

100% - I think here we diverge as this is more a GitHub thing. Thank you for the review, @dipikabh

**Comment by Elenore23** [comment]
> Adds spotlight contributor markdown file and a pull request template to use when nominating someone for a particular role.
> 
> fix #263
>

**Pull request merged**